debian/control

```diff
@@ -0,0 +1,42 @@
+Source: srt
+Priority: optional
+Maintainer: derfel <code@derfel.net>
+Build-Depends: debhelper (>= 9), tcl, pkg-config, cmake, libssl-dev
+Standards-Version: 3.9.8
+Section: libs
+Homepage: https://github.com/Haivision/srt
+
+Package: libsrt-dev
+Section: libdevel
+Architecture: any
+Multi-Arch: same
+Depends: libsrt1 (= ${binary:Version}), ${misc:Depends}
+Description: SRT - Secure, Reliable, Transport
+ Secure Reliable Transport (SRT) is a proprietary transport
+ technology that optimizes streaming performance across
+ unpredictable networks, such as the Internet.
+ .
+ This package contains development files for libsrt1
+
+Package: libsrt1
```

Comment:
libsrt1 -> libsrt?

Reply:
Debian policy requires that the package name match exactly the soname including the version:

Ref: https://www.debian.org/doc/debian-policy/ch-sharedlibs.html#run-time-shared-libraries